components/aws/sagemaker/train/src/train.py

```diff
@@ -21,7 +21,7 @@ def create_parser():
   
   parser.add_argument('--job_name', type=str.strip, required=False, help='The name of the training job.', default='')
   parser.add_argument('--role', type=str.strip, required=True, help='The Amazon Resource Name (ARN) that Amazon SageMaker assumes to perform tasks on your behalf.')
-  parser.add_argument('--image', type=str.strip, required=True, help='The registry path of the Docker image that contains the training algorithm.', default='')
+  parser.add_argument('--image', type=str.strip, help='The registry path of the Docker image that contains the training algorithm.', default='')
```

Comment:
For 1p algorithm, this is not required ?

Reply:
Correct. The code paths for 1P algorithms is you provide an "algorithm name" and then it does the lookup using the SageMaker SDK.